src/test/java/gov/va/viers/cdi/emis/MilitaryInfoTests.java

```diff
@@ -15,7 +20,9 @@
     private MilitaryInfoClient client;
 
     @Test
-    public void tryIt() {
-        assertThat(client.getMilitaryServiceEligibilityResponse("6001010072", "EDIPI")).isEqualTo(null);
+    public void getMilitaryServiceEligibilitySuccess() {
+        JAXBElement<gov.va.viers.cdi.emis.requestresponse.v2.EMISmilitaryServiceEligibilityResponseType> response =
+            client.getMilitaryServiceEligibilityResponse("6001", "EDIPI");
+        assertThat(Optional.ofNullable(response).map(r->r.getValue()).map(v->v.getESSError()).orElse(null)).isNotNull();
```

Comment:
So this looks pretty smart, but I don't think I like it in terms of what it's testing. It looks like the idea is go over every entry in the object that our MilInfo client unmarshals into and check that there's some existent ESS Error. 

I'd rather this checked for one or two key fields of the MilitaryServiceEligibilty entries passed back (since this is a Success test) to confirm that they match what our request asked for/what our mocked VADIR response should provide.